solutions/client-landing-zone/client-folder/firewall-policy/rules/lb-health-checks.yaml

```diff
@@ -22,11 +23,12 @@ metadata:
     config.kubernetes.io/depends-on: compute.cnrm.cloud.google.com/namespaces/client-name-networking/ComputeFirewallPolicy/client-name-client-folder-fwpol # kpt-set: compute.cnrm.cloud.google.com/namespaces/${client-name}-networking/ComputeFirewallPolicy/${client-name}-client-folder-fwpol
 spec:
   action: "goto_next"
-  description: "Goto next for LB health checks"
+  description: "Delegate to host project, ingress LB health checks traffic (firewall) from IP ranges 35.191.0.0/16, 130.211.0.0/22  to VPC resources in shared VPC network"
```

Comment:
Not sure if a linter will catch it, but there's a double space between the IP and to: `130.211.0.0/22  to VPC`